core/base/inc/ROOT/RConfig.h

```diff
@@ -579,5 +579,11 @@
   #define R__likely(expr) expr
 #endif
 
+// Setting this define causes ROOT to keep statistics about memory buffer allocation
+// time within the TTree.  Given that this is a "hot-path", we provide a mechanism
+// for enabling / disabling this at compile time by developers; default is disabled.
+#ifndef R__TRACK_BASKET_ALLOC_TIME
```

Comment:
It may make more sense to put this in TBasket.h

Reply:
It would be convenient, but this is used in multiple headers (`TTree.h`, `TBasket.h`) that have no common include besides `RConfig.h`.

While it's annoying for core IO developers investigating memory allocation costs, it seems a pretty narrow optimization.

Reply:
>  but this is used in multiple headers (TTree.h, TBasket.h) that have no common include besides RConfig.h.

Fair enough, I did not noticed that ...